playroom.config.js

```diff
@@ -1,4 +1,51 @@
 const path = require('path');
+const webpack = require('webpack');
+
+const getMediaQueriesConfig = () => {
+    const impossibleSize = 999999;
+
+    if (process.env.FORCE_MOBILE) {
+        return {
+            tabletMinWidth: impossibleSize,
+            desktopMinWidth: impossibleSize,
+            largeDesktopMinWidth: impossibleSize,
+            desktopOrTabletMinHeight: impossibleSize,
+        };
+    }
+    if (process.env.FORCE_DESKTOP) {
+        return {
+            tabletMinWidth: 0,
+            desktopMinWidth: 0,
+            largeDesktopMinWidth: impossibleSize,
+            desktopOrTabletMinHeight: 0,
+        };
+    }
+    return {
+        desktopOrTabletMinHeight: 0,
+    };
+};
+
+const getOutputPath = () => {
+    if (process.env.FORCE_MOBILE) {
+        return './public/playroom-mobile/';
+    }
+    if (process.env.FORCE_DESKTOP) {
+        return './public/playroom-desktop/';
+    }
+    return './public/playroom/';
+};
```

Comment:
maybe this file could include a comment explaining the reasons of this, as stated in the PR description